<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Open_Form</class>
 <widget class="QWidget" name="Open_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Открыть</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="OkButton">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="SelectCatalog">
     <property name="text">
      <string>Выбрать каталог</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Пересечение по вертикале:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Max:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="MaxVertycalCrossing">
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Min:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="MinVertycalCrossing">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Пересечение по горизонтале:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Max:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="MaxHorizontalCrossing">
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="value">
          <number>127</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Min:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="MinHorizontalCrossing">
         <property name="maximum">
          <number>250</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Тип панорамы:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="Panorama2D">
            <property name="text">
             <string>2D</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="Panorama3D">
            <property name="text">
             <string>3D</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Метод сшивки:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QRadioButton" name="MinimalStitching">
            <property name="text">
             <string>Минимальная разность изображений</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="GradientStitching">
            <property name="text">
             <string>Градиентный метод</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="CatalogPath">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
